shared/haml/user_header.haml

```diff
@@ -120,6 +146,31 @@
       });
     });
 
+    function hideCreateOptions() {
```

Comment:
We should move this to a standalone .js file, even if we are going to inline it for performance reasons.